nowcasting_dataset/dataset/batch.py

```diff
@@ -101,9 +102,14 @@ def batch_to_dataset(batch: List[Example]) -> xr.Dataset:
 
             # GSP
             n_gsp_systems = len(example[GSP_ID])
-            one_dateset["gsp_yield"] = xr.DataArray(
+            one_dateset[GSP_YIELD] = xr.DataArray(
```

Comment:
I'm now about to show my own confusion about `xarray` :)   This comment is as much a question as a code review suggestion!

Is `example[GSP_YIELD]` a `pd.DataFrame` with a `pd.DatetimeIndex` as its index?

If so, then I think the coords for the `time_30` dimension in `one_dateset[GSP_YIELD]` will be an array of _datetimes_ and so won't align with the array of _integer_ coords for the `time_30` dimension in `one_dataset[GSP_DATETIME_INDEX]`.  If that makes sense?!?  (I had to refer to [`xarray`'s documentation](http://xarray.pydata.org/en/stable/user-guide/terminology.html) to remind myself of the distinction between coords and dimensions!)

On the other hand, if `example[GSP_YIELD]` is a numpy array (or a `pd.DataFrame` with an integer index) then all is fine!

Which also makes me think that I should've got into the habit of commenting exactly what type of object each field in `example` is :)

Reply:
Its a numpy array, so I'll add that into the documentation